### PR TITLE
Add home page animations

### LINKED ISSUE #23: [c.c 19-32] Add animations to c.c home page
- [x] Add animations
- [x] Add links to the he logos in the `Product` section
- [x] Add more space after the last line of text in the footer

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Add home page animations

## QA
- Go to [http://0.0.0.0:8002/](http://0.0.0.0:8002/) and make sure there are animations on the page.
- Please test it in multiple browsers.

## Issue

Fixes #23

### CODE REVIEW

File: static/sass/_pattern_footer.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@
     color: $color-mid-light;
     margin-top: -4rem;
     padding-top: 6rem;
+    padding-bottom: 2.5rem;
```
Comment (sowasred2012) [inline static/sass/_pattern_footer.scss]: This line should be above `padding-top`

File: static/sass/_pattern_animations.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
+@mixin canonical-u-animations {
+  .u-animation--slide-from-bottom {
+    opacity: 0;
+    animation: slide-from-bottom map-get($animation-duration, sleepy) linear
```
Comment (sowasred2012) [inline static/sass/_pattern_animations.scss]: missed this yesterday - `animation` should appear before `opacity` in these three classes

**Comment by waynecrosby** [comment]
The spacing between the product boxes has gone large again, the first set are ok but the ones below aren't. Also, is there a way to take over the scroll a bit with some inertia so that as you scroll down a section and near the end it almost snaps to the next section without being too aggressive and annoying?
Can you also add some padding to the bottom of the Partner section to extend the white down a bit.

**Comment by solazio** [comment]
> The spacing between the product boxes has gone large again, the first set are ok but the ones below aren't. Also, is there a way to take over the scroll a bit with some inertia so that as you scroll down a section and near the end it almost snaps to the next section without being too aggressive and annoying?
> Can you also add some padding to the bottom of the Partner section to extend the white down a bit.

@waynecrosby I fixed the spacing and added some additional padding to the bottom of partners section. For the scrolling effect I created a new issue https://github.com/canonical-web-and-design/web-squad/issues/1532

**Pull request merged**